simulator/virtual_machine.go

```diff
@@ -404,8 +407,8 @@ func (vm *VirtualMachine) genVmdkPath() (string, types.BaseMethodFault) {
 			}
 		}
 
-		f.Close()
-		os.Remove(f.Name())
+		_ = f.Close()
+		_ = os.Remove(f.Name())
 
```

Comment:
Just curious, what is the benefit assign return value to `_` here?

Reply:
It surpresses warnings from the go 'errcheck' program. Normally you should check/propagate errors